packages/features/ee/payments/api/webhook.ts

```diff
@@ -155,7 +162,11 @@ async function handlePaymentSuccess(event: Stripe.Event) {
 
   await prisma.$transaction([paymentUpdate, bookingUpdate]);
 
-  await sendScheduledEmails({ ...evt });
+  if (!isConfirmed && !eventTypeRaw?.requiresConfirmation) {
```

Comment:
Can we have a comment here as to what we are trying to do? They way I see it it's handling confirmation for a booking that doesn't need confirmation